src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,28 @@
 package practice;
 
-public class CandidateValidator {
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
     //write your code here
+    private static final int MINIMAL_AGE = 35;
+    private static final String NATIONALITY = "Ukrainian";
+    private static final int LIVED_TEN_YEARS = 10;
```

Comment:
Make constants more informative:
```suggestion
    private static final String REQUIRED_NATIONALITY = "Ukrainian";
    private static final int MINIMUM_RESIDENCY_YEARS = 10;
```